From f9a0d68fd3d21ddeb008e09ed1f223e8aa3774d4 Mon Sep 17 00:00:00 2001
From: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
Date: Thu, 4 Apr 2019 17:44:46 +0300
Subject: [PATCH 148/235] arm64: dts: lx2160a: add interrupt property for
 aquantia phy

Add interrupt property for Aquantia AQR107 ethernet phy, currently
working in polling mode.

Signed-off-by: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
index 6d47ed66db05..74175cf08f92 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
@@ -170,14 +170,14 @@
 	aquantia_phy1: ethernet-phy@4 {
 		/* AQR107 PHY - "compatible" property not strictly needed */
 		compatible = "ethernet-phy-ieee802.3-c45";
+		interrupts = <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
 		reg = <0x4>;
-		/* Poll mode - no "interrupts" property defined */
 	};
 	aquantia_phy2: ethernet-phy@5 {
 		/* AQR107 PHY - "compatible" property not strictly needed */
 		compatible = "ethernet-phy-ieee802.3-c45";
+		interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
 		reg = <0x5>;
-		/* Poll mode - no "interrupts" property defined */
 	};
 };
 
-- 
2.17.1

